test/pkg/testconfig/testconfig.go

```diff
@@ -604,6 +604,7 @@ func CreatePtpConfigWPCGrandMaster(policyName string, nodeName string, ifList []
 	// Sleep for a second to allow previous label on the same node to complete
 	time.Sleep(time.Second)
 	_, err = nodes.LabelNode(nodeName, pkg.PtpClockUnderTestNodeLabel, "")
```

Comment:
should this be deleted? This looks very much like the line caused the CI failures last time.